src/toil/lib/toilDocker.py

```diff
@@ -0,0 +1,213 @@
+"""
+    Module for calling Docker. Assumes `docker` is on the PATH.
```

Comment:
Do we still need `docker` on the PATH? I suspect not.

Reply:
Good point.  I need to update the docstrings.